server/routes/api.php

```diff
@@ -2,27 +2,13 @@
 
 use Illuminate\Http\Request;
 
-/*
-|--------------------------------------------------------------------------
-| API Routes
-|--------------------------------------------------------------------------
-|
-| Here is where you can register API routes for your application. These
-| routes are loaded by the RouteServiceProvider within a group which
-| is assigned the "api" middleware group. Enjoy building your API!
-|
-*/
+Route::group([
+    'prefix' => 'auth'
+], function () {
 
-Route::group(['prefix' => 'post'], function() {
-    Route::get('/', 'PostController@index');
-    Route::get('{id}', 'PostController@show');
-    Route::post('/', 'PostController@store');
-    Route::put('{id}', 'PostController@update');
-    Route::delete('{id}', 'PostController@destroy');
+    Route::post('login', 'AuthController@login');
+    Route::post('logout', 'AuthController@logout');
+    Route::post('refresh', 'AuthController@refresh');
```

Comment:
why me and refresh are post requests? do they have body?

Reply:
I had not used them at that moment, this routes are default routes advised by 'Tymon/JWT' package. I will use them later. Next time after committing something I will delete such things.